### PR TITLE
fixing summary and description generation

### PR DESCRIPTION
**Pull request by sarunassp**

### CODE REVIEW

File: src/SwashBuckle.AspNetCore.MicrosoftExtensions/Extensions/MetadataAttributeExtensions.cs [hunk 1]
```diff
@@ -14,7 +14,19 @@ internal static class MetadataAttributeExtensions
             if (attribute.Summary != null)
                 yield return new KeyValuePair<string, object>(Constants.XMsSummary, attribute.Summary);
             if (attribute.Description != null)
-                yield return new KeyValuePair<string, object>(Constants.XMsDescription, attribute.Description);
+                yield return new KeyValuePair<string, object>(Constants.Description, attribute.Description);
+        }
+
+        internal static IEnumerable<KeyValuePair<string, object>> GetSwaggerOperationExtensions (this MetadataAttribute attribute)
```
Comment (BenasJacikas) [inline src/SwashBuckle.AspNetCore.MicrosoftExtensions/Extensions/MetadataAttributeExtensions.cs]: Specify in comment why operation description differs from other item descriptions and throw in a link to documentation

**Review by BenasJacikas — CHANGES_REQUESTED** [review]

File: src/SwashBuckle.AspNetCore.MicrosoftExtensions/Extensions/MetadataAttributeExtensions.cs (hunk 1 above)
Comment (sarunassp) [reply src/SwashBuckle.AspNetCore.MicrosoftExtensions/Extensions/MetadataAttributeExtensions.cs]: Only the summary summary differs
"summary" for operations
"x-ms-summary" for parameters
https://flow.microsoft.com/en-us/documentation/customapi-how-to-swagger/

**Comment by BenasJacikas** [comment]
Change readme.md to reflect changes

File: src/SwashBuckle.AspNetCore.MicrosoftExtensions/Constants.cs [hunk 2]
```diff
@@ -5,11 +5,12 @@ internal static class Constants
...
         internal const string XMsVisibility = "x-ms-visibility";
         internal const string XMsSummary = "x-ms-summary";
-        internal const string XMsDescription = "x-ms-description";
+        internal const string Description = "description";
         internal const string XMsDynamicValueLookup = "x-ms-dynamic-values";
         internal const string XMsDynamicSchemaLookup = "x-ms-dynamic-schema";
         internal const string Parameter = "parameter";
         internal const string XMsCapabilities = "x-ms-capabilities";
         internal const string FilePicker = "file-picker";
+		internal const string Summary = "summary";
```
Comment (BenasJacikas) [inline src/SwashBuckle.AspNetCore.MicrosoftExtensions/Constants.cs]: Remove tabs

**Review by BenasJacikas — APPROVED** [review]

**Pull request merged**